baremetal/metal3data_manager.go

```diff
@@ -275,7 +276,7 @@ func (m *DataManager) createSecrets(ctx context.Context) error {
 	// The NetworkData secret must be created
 	if apierrors.IsNotFound(networkDataErr) {
 		m.Log.Info("Creating Networkdata secret")
-		networkData, err := renderNetworkData(m3dt, bmh, poolAddresses)
+		networkData, err := renderNetworkData(m3dt, m3m, capiMachine, bmh, poolAddresses)
```

Comment:
nit:
```suggestion
		networkData, err := renderNetworkData(m3dt, m3m, machine, bmh, poolAddresses)
```
for consistency

Reply:
The observation is valid, but this inconsistency is not introduced this MR.

There could surely be done something more consistent terminology concerning variable names etc., but this should IMHO go into a separate MR.

For example, a good starting point could be to make the symbolic identifiers

```go
const (
	m3machine = "metal3machine"
	host = "baremetalhost"
	capimachine = "machine"
)
```

less deviant and then make the variable terminology  more consistent.